### ISSUE TITLE
Add link to api page

### ISSUE THREAD

**Issue by peterfpeterson** [mantidproject/webapp#5]
The page http://reports.mantidproject.org/api/ is missing a link to the feature usages. Add a line to [`views.py`](https://github.com/mantidproject/webapp/blob/master/wsgi/openshift/services/views.py#L208) like

``` python
'feature':  reverse('feature',  request=request, format=format)
```

**Comment by micah-raney** [comment]
Found through some digging that the model being referred to is called "featureusage-list". 

`'feature':  reverse('featureusage-list',  request=request, format=format)`

the right place for this code is in api_root( ).

**Comment by micah-raney** [comment]
Fix implemented in 178c2a28ca9d8a124b5037629e84a5031c010f69
